src/onnx/parse_qlinearunary.cpp

```diff
@@ -0,0 +1,299 @@
+{
+    if(args.size() < 4)
+        MIGRAPHX_THROW(opd.op_name + ": missing inputs");
+
+    const auto& in_x = args[0];
+
+    auto sh_x   = in_x->get_shape();
+    auto type_x = sh_x.type();
+    if(type_x != migraphx::shape::int8_type and type_x != migraphx::shape::uint8_type)
+        MIGRAPHX_THROW(opd.op_name + ": unsupported input type");
+}
+
+template <class F>
+instruction_ref parse_qlinearunary(const onnx_parser::node_info& info,
+                                   const std::vector<instruction_ref>& args,
+                                   F f)
+{
+    // X
+    const auto& in_x         = args[0];
+    const auto& in_scale_x   = args[1];
+    const auto& in_zero_pt_x = args[2];
+
+    auto dquant_x = bcast_qdq_instr("dequantizelinear", in_x, in_scale_x, in_zero_pt_x, info);
+
+    // Y = (op(dequantizelinear(x))
+    auto y = f(dquant_x);
+
+    const auto& in_scale_y = args[3];
+
+    // zero_pt for Y is supplied as the last optional argument..
+    if(args.size() == 5)
+        return (bcast_qdq_instr("quantizelinear", y, in_scale_y, args[4], info));
+
+    // if no zero_pt: just broadcast the scale..
+    auto bcast_scale_sigm = bcast_scalar_instr(y->get_shape(), in_scale_y, info);
+    return (info.add_instruction(migraphx::make_op("quantizelinear"), y, bcast_scale_sigm));
+}
+
+/*
+ *********************************************************************************
+ *  Reference: see QLinearSigmoid in                                             *
+ *  https://github.com/microsoft/onnxruntime/blob/main/docs/ContribOperators.md  *
+ *********************************************************************************
+
+com.microsoft.QLinearSigmoid
+QLinearSigmoid takes quantized input data (Tensor), and quantize parameter for output, and produces
+one output data (Tensor) where the function f(x) = quantize(Sigmoid(dequantize(x))), is applied to
+the data tensor elementwise. Where the function Sigmoid(x) = 1 / (1 + exp(-x))
+
+Version
+This version of the operator has been available since version 1 of the 'com.microsoft' operator
+set.
+
+Inputs (4 - 5)
+X : T
+Input tensor
+
+X_scale : tensor(float)
+Input X's scale. It's a scalar, which means a per-tensor/layer quantization.
+
+X_zero_point (optional) : T
+Input X's zero point. Default value is 0 if it's not specified. It's a scalar, which means a
+per-tensor/layer quantization.
+
+Y_scale : tensor(float) Output Y's scale. It's a scalar, which means
+a per-tensor/layer quantization.
+
+Y_zero_point (optional) : T Output Y's zero point. Default value is
+0 if it's not specified. It's a scalar, which means a per-tensor/layer quantization.
+
+Outputs
+Y : T
+Output tensor
+
+Type Constraints
+T : tensor(uint8), tensor(int8)
+Constrain input and output types to 8 bit tensors.
+
+*/
+
+struct parse_qlinearsigmod : op_parser<parse_qlinearsigmod>
+{
+    std::vector<op_desc> operators() const { return {{"QLinearSigmoid", "sigmoid"}}; }
```

Comment:
This should be `{"QLinearSigmoid", "sigmoid"}, {"QLinearLeakyRelu", "leaky_relu"}}`.